_includes/v21.1/orchestration/start-cockroachdb-helm-insecure.md

```diff
@@ -94,5 +112,5 @@ The CockroachDB Helm chart is undergoing maintenance for compatibility with Kube
     ~~~
 
 {{site.data.alerts.callout_success}}
-The StatefulSet configuration sets all CockroachDB nodes to log to `stderr`, so if you ever need access to a pod/node's logs to troubleshoot, use `kubectl logs <podname>` rather than checking the log on the persistent volume.
+The StatefulSet configuration sets all CockroachDB nodes to log to `stderr`, so if you ever need access to logs for a pod or node, use `kubectl logs <podname>` rather than checking the log on the persistent volume.
```

Comment:
```suggestion
The StatefulSet configuration sets all CockroachDB nodes to log to `stderr`, so if you ever need access to logs for a pod, use `kubectl logs <podname>` rather than checking the log on the persistent volume.
```